src/main/java/racing/model/CarRacingGame.java

```diff
@@ -0,0 +1,52 @@
+package racing.model;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+
+public class CarRacingGame {
+
+    private final List<Car> cars = new ArrayList<>();
+    private final int numberOfMove;
+    private int movingCount = 0;
+
+    public CarRacingGame(List<String> carNames, int numberOfMove) {
+        for (String name : carNames) {
+            Car car = new Car(name);
+            cars.add(car);
+        }
```

Comment:
```suggestion
        this.cars = carNames.stream()
                .map(Car::new)
                .collect(Collectors.toList());
```
요 부분도 스트림을 만들어줄 수 있겠네요ㅎㅎ